### PR TITLE
[MacOS] Column width edge case fixes

### PR DESCRIPTION
**Pull request by blue-stone99**
## Sample Card
image
image
image

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift [hunk 1]
```diff
@@ -52,32 +52,29 @@ class ACRColumnView: ACRContentStackView {
...
-        minWidthConstraint.priority = .defaultHigh
-        minWidthConstraint.isActive = true
-    }
-    
     override func addArrangedSubview(_ subview: NSView) {
+        // Activate min Width constraint only if column has a non-image subview
+        // Because - column can be empty                        -> should have zero content size
+        //         - image can have dimensions less than 30 pts -> should match image content size
+        let contentStackView = subview as? ACRContentStackView
+        if !(contentStackView?.arrangedSubviews.first is ACRContentHoldingView || subview is ACRContentHoldingView) {
```
Comment (manujai) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift:74]: I will probably change the name `ACRContentHoldingView` to `ACRImageHoldingView` or something in my next PR for better context. 
Because the name does not seem straightforward here in this case.

**Review by manujai — APPROVED** [review]
